src/main/java/frc/robot/subsystems/ShooterArmSubsystem.java

```diff
@@ -101,8 +101,8 @@ public boolean getSwitch(){
     return this.limitSwitch.get();
   }
 
-  public void moveArmByVoltage(){
-    m_ArmMotor.setVoltage(ArmStaticMovmentSpeed);
+  public void movementArmToReset(){
```

Comment:
change to a func the gets the speed in the () and use the resetSpeed in the command

Reply:
did it